readme.md

```diff
@@ -126,12 +126,22 @@ You may also pass a sub-application, which _must_ be accompanied by a `base` pat
 Please see [`Middleware`](#middleware) and [Express' middleware examples](http://expressjs.com/en/4x/api.html#middleware-callback-function-examples) for more info.
 
 
-### parse(req)
+### parse(req, toDecode=true)
 
 Returns: `Object` or `undefined`
 
 As of `v0.5.0`, this is an alias of the [`@polka/url`](/packages/url) module. For nearly all cases, you'll notice no changes.
 
+One important difference is that URL decoding is enabled by default in polka but it is disabled by default in the `@polka/url` module. You can disable the URL decoding if required,
```

Comment:
This should call out that it's only enabled by default in `@next`

Reply:
No need since this is the `next` branch, implied. Will only appear on any `@next` installations